### DISCUSSION TITLE
IPF color VTK files when exporting

### DISCUSSION THREAD

**Discussion by mvailhe** [neperfepx/neper#888] [Q&A] [answered]
I wanted to follow up on discussion #136 and see if there was any update on exporting a VTK file that includes IPF coloring. 

You can export IPF coloring using the code below for a .png file:

`neper -V n200.tess -datacellcol ori -datacellcolscheme ipf -print img10`

However, if modified for a .vtk file there is no option in paraview where IPF coloring can be viewed and appear to be the same output

`neper -V n200.tess,n200.msh -imageformat vtk -print vtk1`

`neper -V n200.tess,n200.msh -datacellcol ori -datacellcolscheme ipf -imageformat vtk -print vtk2`

![vtk1](https://github.com/user-attachments/assets/78cd599d-4a94-4e3c-9206-8ee69a6297a9)
![vtk2](https://github.com/user-attachments/assets/e334e944-2b23-4d89-8ca8-cb254d5a5f21)

**Comment by rquey** [comment, marked as answer]
If you can tell me the syntax to define cell RGB colors in the vtk file, this is something I can add.
